src/lib/apiTypesV2/HttpInfo.cpp

```diff
@@ -91,4 +94,52 @@ std::string HttpInfo::toJson()
   return jh.str();
 }
 
+
+
+/* ****************************************************************************
+*
+* HttpInfo::fill -
+*/
```

Comment:
Citing myself :)

> In my opinion the decission should be done by the author [in .cpp files]

You are the author of this part of the code. So, it's up to you.